.github/workflows/main.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     outputs:
       NPM_VERSION: ${{ steps.version.outputs.NPM_VERSION }}
       PUBLISH_TAG: ${{ steps.version.outputs.PUBLISH_TAG }}
+      IMAGE_SUFFIX: ${{ steps.version.outputs.IMAGE_SUFFIX }}
```

Comment:
Shouldn't IMAGE_PATH be added here?

Reply:
I tried that way, but it came as blank in the `Build & Push` step.
So I shifted the `IMAGE_PATH` creation to the `Slug` step & it worked
So we don't need to add `IMAGE_Path` here.